src/pages/no-quizzes-page/no-quizzes.tsx

```diff
@@ -1,7 +1,43 @@
+import { Button } from "primereact/button";
+import { Card } from "primereact/card";
 import React from "react";
 
+import "./no-quizzes.scss";
+
 interface NoQuizzesProps {}
 
 export const NoQuizzes: React.FC<NoQuizzesProps> = ({}) => {
-  return <div></div>;
+  const footer = (
```

Comment:
Since this variable is of type JSX it is good practice to name it as a component capitalized. In this case it would be `Footer`